core/src/main/java/io/questdb/griffin/engine/functions/groupby/ApproxCountDistinctIPv4GroupByFunction.java

```diff
@@ -41,16 +41,16 @@ public class ApproxCountDistinctIPv4GroupByFunction extends LongFunction impleme
     private static final long NULL_VALUE = -1;
 
     private final Function arg;
-    private final HyperLogLog hllA;
-    private final HyperLogLog hllB;
+    private HyperLogLog hllA;
```

Comment:
To tolerate `null`s for such special cases (state fields that are initialized later) in the `FunctionCloneFactory` we could introduce a runtime annotation and mark these fields with it. This way we know that we can safely set the field to null when cloning. WDYT?